strider/server.py

```diff
@@ -161,10 +161,6 @@ def get_finished_query(
 
     # strip categories from node bindings
```

Comment:
This comment should be removed.

Reply:
fe0455dc3250e82c05c401530b1061ccbb8f70ac